react-native-libraries.json

```diff
@@ -9453,5 +9453,12 @@
     "examples": ["https://github.com/pushpender-singh-ap/react-native-scanner/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/frw/react-native-ssl-public-key-pinning",
+    "npmPkg": "react-native-ssl-public-key-pinning",
```

Comment:
Hello @frw, thank you for adding new library to the directory! 🙏 

```suggestion
```

The package name field can be skipped, since it matches repository name, beside that LGTM! 👍

Reply:
Thanks @Simek, I've removed it!